public/style.css

```diff
@@ -0,0 +1,211 @@
+/* Start preset */
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
```

Comment:
what is this styling for?

Reply:
It's a general reset for the elements, we found that developers do more than resetting the padding and margins, so we used it!